index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+  <script src="https://unpkg.com/boxicons@2.1.4/dist/boxicons.js"></script>
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <i class='bx bx-menu bx-sm'></i>
+    </nav>
+</header>
+  <section class="headline">
+    <div class="container">
+      <h1>Hey there. <br>
+        I’m Tsige.</h1>
+      <h1 class="green">I’m a software developer! </h1S></div>
```

Comment:
- [ ] You did a great job changing the greeting to `h1` for SEO reasons. :+1: Another best practice when using `h1` element is that we should only have one `h1` element per page. Please use another `hx` element such as `h2` for the text `I’m a software developer!` to make sure we are following best practices. 😎️ Please check this [resource](https://www.reliablesoft.net/h1-tag/#tip7) to get more insight about best practices regarding the `h1` element.